src/main/java/com/uid2/admin/Main.java

```diff
@@ -309,6 +309,9 @@ public void run() {
             DataStoreMetrics.addDataStoreMetrics("enclaves", enclaveIdProvider);
             DataStoreMetrics.addDataStoreMetrics("salt", saltProvider);
             DataStoreMetrics.addDataStoreMetrics("partners", partnerConfigProvider);
+            DataStoreMetrics.addDataStoreMetrics("service_link", serviceLinkProvider);
+            DataStoreMetrics.addDataStoreServiceLinkEntryCount("snowflake", serviceLinkProvider, serviceProvider);
```

Comment:
I think we should make this method specific to Snowflake, I don't see how it could be used for other service links

Therefore we don't need to pass in the service name, just hardcode it (with a constant) inside the method and make the method name Snowflake specific